newm-server/src/main/kotlin/io/newm/server/features/song/SongRoutes.kt

```diff
@@ -69,7 +76,18 @@ fun Routing.createSongRoutes() {
                         }.sumOf { it.amount }
                     song.copy(earnings = allEarningsAmount)
                 }
-                respond(songsAndEarnings)
+                if (songFilters.sortedBy == "earnings") {
```

Comment:
This is the meat of the PR. The rest is the major exposed update.  I attempted to update KTOR at the same time to version 3, but we need to wait until Koin supports that version of KTOR.